Update of PR #503 (#519)
cf-convention/cf-convention.github.io: projects-activities.md

<!-- ***ISIMIP links to CF needs to be established - seems to use basic CMIP style metadata*** -->

> ## Satellite and remote sens found only in commented text:ing data

Reviewer: ```suggestion
## Satellite and remote sensing data
```

Not sure what happened here - looks like some text has accidentally been added?

Author: Ooops, probably fat fingers or too sensitive mouse pad...

[Copernicus CDS](https://cds.climate.copernicus.eu) provides comprehensive climate data and services. Depending on data type several formats are used, including netCDF/CF. External data providers are requested to follow a specified data format that builds on CF (see this [link](https://confluence.ecmwf.int/display/COPSRV/%28version+C3S-0.3%29+Guide+to+NetCDF+encoding+for+C3S+providers) for details).

## Data and metadata standards that build on and extend CF

### WMO Experimental CF-netCDF Profiles

Within the overaching framework of WMO Information System (WIS) extensions to the Climate and Forecast Conventions (CF conventions) for netCDF are developed (WMO-CF extensions, or *profiles*). They aim to reduce the effort involved in specifying data products and ensure interoperability by providing standardized semantics and metadata. The level of standardization provided by the WMO-CF extensions is in many cases greater than what is provided by the CF conventions to satisfy the specific WMO operational needs to support Earth System monitoring and prediction. New netCDF profiles are defined to act as templates for reporting data from different sources.  

#### WMO-CF RADIAL 

> This *profile* should be used for the representation of weather radar and lidar data in the native instrument-centric polar coordinates. The profile is now published in [WMO-No. 1257 Volume VII](https://library.wmo.int/idurl/4/68826). In addition this profile is being extended and updated in [CFRadial2](http://dx.doi.org/10.5065/fy2k-x587).

Reviewer: ```suggestion
This *profile* should be used for the representation of weather radar and lidar data in the native instrument-centric polar coordinates. The profile is now published in [WMO-No. 1257 Volume I](https://library.wmo.int/idurl/4/68826). In addition this profile is being extended and updated in [CFRadial2](http://dx.doi.org/10.5065/fy2k-x587).
```

At least, the link goes to a page quoting 'Volume I: Weather Radar Network Programme Design', so either the number is wrong, or the link not quite right?

Reviewer: Excellent. Thanks for the quick update. I am marking as resolved and will then merge.

## Arctic Data Center

> Managed by the National Cent found only in commented text:er for Ecological Analysis and Synthesis (NCEAS), this repository adheres to CF conventions for its extensive collection of Arctic research data.

Reviewer: As per the comment above :)

```suggestion
Managed by the National Center for Ecological Analysis and Synthesis (NCEAS), this repository adheres to CF conventions for its extensive collection of Arctic research data.
```

Author: I have removed text fragments in comments that were left from the initial commit.

ESMO | Earth System Modelling and Observations is a core activity of the WCRP
EUMETSAT | European Organisation for the Exploitation of Meteorological Satellites
GCM | Global Climate Model
IMOS | ----- still only in comment

> Input4MIPs | ----- not an acronym ???

Reviewer: I would suggest partial acronyms such as this should still be on the list since e.g. 'Obs' in 'Obs4MIPs' is still abbreviated even if 'Input' isn't here, where '4' is an abbreviation in itself, but it won't block my approval of the PR.

Author: I am a bit reluctant to include them because they do not appear anywhere else in the text except in connection where they are explained anyway.

Reviewer: Oh! In that case it is all very sensible then. I didn't realise that context (I did search on those keywords in the document but I didn't clock they were explained in the relevant places). Yes, no need to be overkill! Thanks Lars.
